test_site/sub_site/site.json

```diff
@@ -0,0 +1,16 @@
+{
+  "baseUrl": "/sub_site",
+  "pages": [
+    {
+      "src": "index.md",
+      "title": "Hello World"
+    }
+  ],
+  "ignore": [
+    "_site/*",
+    "site.json"
```

Comment:
could we ignore `*md` files as well? I believe our test does not need to check for copied md files

Reply:
Sure, will add the required changes.